src/less/mainNav.less

```diff
@@ -136,68 +142,42 @@ main-nav {
 }
 
 .main-nav-menu-items {
-  list-style: none;
-  padding: 0;
+  height: 100%;
+  margin-right: -15px;
   overflow-y: auto;
   overflow-x: hidden;
-  height: 100%;
+  padding: 0;
   position: relative;
-  margin-right: -15px;
+  list-style: none;
 
   li {
     .transition(background-color .2s, width .2s, height .2s;);
     border-bottom: 1px solid @gray-900;
     width: @main-nav-width-sm;
     height: 44px;
 
-    &:hover {
-      background-color: @main-nav-bg-hover;
-    }
-
     &.open {
-      background-color: @main-nav-bg-selected;
+      .btn {
+        background-color: @main-nav-active-bg;
+      }
 
       a,
       .btn {
-        color: #fff;
+        color: @main-nav-link-text-active;
 
         .cyclops-icon use {
-          fill: #fff;
+          fill: @main-nav-link-text-active;
         }
       }
     }
 
     &.selected {
-      background-color: @main-nav-bg-selected;
-      box-shadow: @main-nav-selected-highlight;
-      border: 1px solid @main-nav-selected-highlight;
-
-      .cyclops-icon use {
-        fill: #fff;
-      }
-    }
-
-    .btn {
-      width: 100%;
-      margin: 0;
-      background-color: transparent;
-      color: @main-nav-link-color;
-      text-transform: capitalize;
-
-      &:hover {
-        color: #fff;
-
-        .cyclops-icon {
-          fill: @gray-100;
-        }
-      }
-
-      &:focus,
```

Comment:
Moved setting the background colors to the .btn for the active/focus/hover states https://github.com/CenturyLinkCloud/Cyclops/pull/59/files/9eec78ec184603010b3ed808f8f4699961ac6eeb#diff-acdb910db649d574582b905a2ca0c99aL60